health.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"net/http"
+	"github.com/gorilla/mux"
+	log "github.com/sirupsen/logrus"
+)
+
+func startRestAPI() {
+	router := mux.NewRouter()
+	router.Methods("GET").Path("/health").HandlerFunc(getHealth)
+	router.Use(errorLogger)
+	go func() {
+		log.Fatal(http.ListenAndServe(":9000", router))
```

Comment:
don't we usually put health checks on port 7070 (just for the sake of consistency)?

Reply:
> actually, this just occurred to me: do you really need a fork with changed code?
> As far as I can tell, what you've added does not change the core business logic of burry, it just adds a mode that runs that core business logic periodically.
> If that is really all that was changed, it's maybe more convenient (and more easy to maintain) to indeed create a separate repository that just has a Dockerfile and a script that periodically runs burry+sleep in a loop.

The one thing it does differently (and which is hard to code in a script) is polling the zookeeper periodically and only writing a S3 backup when there are any changes.

Reply:
changed